docs/conf.py

```diff
@@ -260,12 +260,17 @@
         "Community": {
             "Contributing": {
                 "description": "Learn how to contribute to the Litestar project",
-                "link": "https://docs.litestar.dev/2/contribution-guide.html",
+                "link": "https://docs.litestar.dev/latest/contribution-guide.html",
                 "icon": "contributing",
             },
             "Code of Conduct": {
                 "description": "Review the etiquette for interacting with the Litestar community",
-                "link": "https://github.com/litestar-org/.github/blob/main/CODE_OF_CONDUCT.md",
+                "link": "https://github.com/litestar-org/.github?tab=coc-ov-file",
+                "icon": "coc",
+            },
+            "Security": {
+                "description": "Learn how we handle security issues in Litestar",
```

Comment:
The wording doesn’t fit here. It’s not a tutorial so "learn how.." doesn’t really make sense